libledger/Ledger.cpp

```diff
@@ -351,11 +351,33 @@ void Ledger::initDBConfig(ptree const& pt)
     {
         m_param->mutableStorageParam().maxRetry = 100;
     }
+    /// set state db related param
+    m_param->mutableStateParam().type = pt.get<std::string>("state.type", "storage");
+
+    // read db config from config eg:mysqlip mysqlport and so on
+    m_param->mutableStorageParam().dbtype = pt.get<std::string>("storage.dbtype", "mysql");

```

Comment:
this config item is unnecessary, we only support mysql

Reply:
actually we can support sqllite,oracle and postgresql if necessary.

Reply:
I know that, but we will not to do that. If we do, we can add this configuration